ddtrace/contrib/django/patch.py

```diff
@@ -734,7 +734,10 @@ def wrap_wsgi_environ(wrapped, _instance, args, kwargs):
 
         from ddtrace.appsec.iast._taint_utils import LazyTaintDict
 
-        return wrapped(*((LazyTaintDict(args[0]),) + args[1:]), **kwargs)
+        return wrapped(
+            *((LazyTaintDict(args[0], origins=(IAST.HTTP_REQUEST_HEADER_NAME, IAST.HTTP_REQUEST_HEADER)),) + args[1:]),
```

Comment:
Question: Can the wsgi environ have a size of 1?  If so, this can throw a value error

Reply:
Not sure I understand, so I just verified that in our tests, `len(args) == 1`: the first argument is a dict with the wsgi environ, no errors raised. Also checked with that dict being one entry only, and no errors at this point.